README.md

```diff
@@ -133,6 +134,32 @@ module.exports = {
 };
 ```
 
+### `crossOrigin`
```

Comment:
I am fine with this PR, but let's do better name for this option, `crossOrigin` sounds misleading even for me although I understand that we are trying to solve this

Reply:
Maybe `originPath`?

Reply:
Thanks change runtime logic, so I think we should point it out

Reply:
Maybe `workerChunkUrl`? Hard for me too, here we have two changes, using `import-scripts` and pass it like URL, it can be misleading with `inline`, I think ideally we should have `chunkLoadingType: 'inline-fallback' | 'inline-no-fallback' | 'import-scripts'` and set `publicPath` to `Absolute` URL, in runtime we should do `publicPath` + `filename`

Reply:
I understand the need to use `publicPath` but there's a need to have different public paths for different assets. Maybe I should test whether I can change `publicPath` on the fly before importing worker. Also why do you refer to worker as "chunk" instead of "script"? Maybe have something like `scriptType: "inline-fallback" | "inline-no-fallback" | "import-scripts"`?